components/ImageUploadForm/index.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import useSWR from "swr";
+
+function ImageUploadForm({ onAddImage }) {
+  const { mutate } = useSWR("/api/images");
+
+  const [uploadStatus, setUploadStatus] = useState("");
+  const [error, setError] = useState(undefined);
+
+  async function submitImage(event) {
+    event.preventDefault();
+    setUploadStatus("Uploading...");
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    try {
+      const response = await fetch("/api/upload", {
+        method: "POST",
+        body: formData,
```

Comment:
```suggestion
        body: data,
```
Please check this before you merge :)

Reply:
Please test this:

console.log(data) 
-> above line 16

console.log(image) 
-> in line 28 
Compare the two logs and see if they log out a similar or the same thing. If yes, please adjust your `const newImage` keys, to follow the same object so ideally it would look like this:


```
const newImage = {
        id: image.public_id,
        image: image.secure_url,
        theme: image.theme,
        description: image.description,
        username: image.username,
        isCloudinaryImage: true,
      };
 ```
 Is this possible?
 If no, we can look over it together :)

Reply:
I tried it but it then doesn't work as it should. On the "my art" page the uploaded image is not displayed anymore and on the details page the input values like "theme", "description" and "username" aren't shown as well.